src/datepicker/datepicker-input.ts

```diff
@@ -141,6 +142,22 @@ export class NgbInputDatepicker implements OnChanges,
    */
   @Output() navigate = new EventEmitter<NgbDatepickerNavigateEvent>();
 
+  @Input()
+  get disabled() {
+    return !!this._disabled;
+  }
+  set disabled(value: any) {
+    let isDisabled = value != null && `${value}` !== 'false';
+
+    if (this._disabled !== isDisabled) {
```

Comment:
honestly I wouldn't care checking this. I'm sure the `NgbDatepickerService` would not set the same value twice